Assets/Scripts/Layers/ObjectLibrary/Windmill.cs

```diff
@@ -0,0 +1,105 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Netherlands3D.T3DPipeline;
+
+public enum WindmillStatus
+{
+    Unknown,
+    Active,
+    Planned,
+    Removed
+}
+
+public class Windmill : MonoBehaviour
```

Comment:
De Twin/Netherlands3D package heeft een Windmills pseudo-package waar alle windmolen dingen al inzitten, inclusief dit script: C:\Unity\Twins\Library\PackageCache\nl.netherlands3d@9e952c6531\Windmills\Runtime\Scripts\Windmill.cs

Reply:
windmill model is aangepast om hier een variant op te zijn